src/test/java/com/stormeye/evaluation/BlockStepDefinitions.java

```diff
@@ -105,8 +105,8 @@ public void aValidErrorMessageIsReturned() {
         final CasperClientException csprClientException = contextMap.get("csprClientException");
 
         assertThat(csprClientException.getMessage(), is(notNullValue()));
-        assertThat(csprClientException.getMessage().toLowerCase().contains(blockErrorMsg), is(true));
-        assertThat(csprClientException.getMessage().toLowerCase().contains(blockErrorCode), is(true));
+        assertThat(csprClientException.getMessage().toLowerCase().contains(blockErrorMsg.toLowerCase()), is(true));
```

Comment:
use the correct matcher for both assertions here eg:
`assertThat(csprClientException.getMessage(), containsStringIgnoringCase(blockErrorMsg));`